website/docs/api/corpus.mdx

```diff
@@ -175,3 +175,68 @@ Yield examples from the data.
 | ---------- | -------------------------------------- |
 | `nlp`      | The current `nlp` object. ~~Language~~ |
 | **YIELDS** | The examples. ~~Example~~              |
+
+## PlainTextCorpus {id="plaintextcorpus",tag="class"}
```

Comment:
We should add `v4` tags to the new additions like this one and the ones below.

Reply:
Added a version attribute to the class.